### PR TITLE
Fix pod span display - prevent certain unwanted stages to become parent stage

### PR DESCRIPTION
**Pull request by wuchang0201**
In podCreate, some stages such as “image_pull_span” may become the parent stage， need to be adjusted

### CODE REVIEW

**Comment by wuchang0201** [comment]
example：
image

File: internal/grafanadi/service/span.go [hunk 1]
```diff
@@ -41,20 +52,48 @@ func ConverSpan2Frame(spans []*storagemodel.Span) model.DataFrame {
 		durationAry = []int64{int64(rootEnd.Sub(rootBegin) / 1000 / 1000)}
 	}
 
+	var opsType string
+	m1 := make(map[string]string, 0)
```
Comment (hnhbwlp) [inline internal/grafanadi/service/span.go:56]: The variable name of m1 needs to be optimized. It makes no sense to define it like this.

File: internal/grafanadi/service/span.go [hunk 2]
```diff
@@ -41,20 +52,48 @@ func ConverSpan2Frame(spans []*storagemodel.Span) model.DataFrame {
...
+			m1[sp.ActionType] = spanId
+			pspanAry = append(pspanAry, rootSpanID)
+			if opsType != "pod_ready_span" && sp.ActionType == "PodCreate" {
+				opsAry[1] = ""
+				durationAry[1] = int64(maxEnd.Sub(minBegin) / 1000 / 1000)
+				startTimeAry[1] = minBegin.UnixNano() / 1e6
+				traceAry = append(traceAry, rootTraceID)
+				spanAry = append(spanAry, spanId)
+
+				pspanAry = append(pspanAry, spanId)
```
Comment (hnhbwlp) [inline internal/grafanadi/service/span.go:87]: Also with the variable name of pspan needs to be optimized.

Comment (wuchang0201) [reply internal/grafanadi/service/span.go]: LGTM

**Review by d3c3mber — APPROVED** [review]

**Review by hnhbwlp — APPROVED** [review]

**Comment by wuchang0201** [comment]
#95

**Pull request merged**